en/docs/install-and-setup/setup/api-controller/advanced-topics/using-dynamic-data-in-api-controller-projects.md

```diff
@@ -2,98 +2,42 @@
-
-Follow the instructions below to initialize an API Project with environment variables.
-
-1.  Create a file `api-env-config.yaml` with the below content.
-
-    ```bash
-    type: api
-    version: v4.0.0
-    data:
-        name: $APINAME
-        context: /petstore/$APIVERSION
-        version: $APIVERSION
-    ```
-
-2.  Export the environment variables with required values.
-
-    ```bash tab="Linux/Mac"
-    export APINAME=Petstore
-    export APIVERSION=1.0.0
-    ```
-
-    ```bat tab="Windows"
-    SET APINAME=Petstore
-    SET APIVERSION=1.0.0
-    ```
+## Add dynamic data to environment configs
 
-3.  Initialize the API Project with the `api-env-config.yaml`.
+To allow easily configuring environment-specific details, by default, the apictl supports an additional parameter file. For more information on using an environment parameter file, see [Configuring Environment Specific Parameters]({{base_path}}/install-and-setup/setup/api-controller/advanced-topics/configuring-environment-specific-parameters). 
 
-    ```
-    apictl init PetstoreProject -d api-env-config.yaml --oas https://petstore.swagger.io/v2/swagger.json
-    ```
+The file supports detecting environment variables during the API (or API Product) import process. You can use the notation `${DEV_URL}` to specify environment variables to any attribute in this file. 
 
-    !!! tip
 
-        Upon successful completion of the above command, the apictl will automatically inject the environment variable values to the API artifact in the API project. Open `PetstoreProject/api.yaml` and check the above values are correctly injected.
+## Initialize API projects with dynamic data
 
-        ```
-        type: api
-        version: v4.0.0
-        data:
-            name: Petstore
-            description: 'This is a sample server Petstore server.  You can find out more about
-                Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For
-                this sample, you can use the api key `special-key` to test the authorization filters.'
-            context: /petstore/1.0.0
-            version: 1.0.0
-        ...
-        ```
+When initializing an API Project, the apictl is capable of detecting environment variables in the default definition file or in the provided custom definition file. Then it will create the primary API configuration file `api.yaml` with the dynamic data based on the environment variables. When executing the `apictl init` command, the CTL automatically injects the values to the API definition. You can use the notations `$ENV_VAR` or `${ENV_VAR}` to specify environment variables to any attribute in this file.
```

Comment:
As discussed, lets stop supporting `$ENV_VAR` format, so we should not mention it as an option. Lets stick with `${ENV_VAR}` inly

Reply:
I have mistakenly committed unnecessary content on the env variables of API definition. Since we do not support injecting env variables to API definition, this content is removed via https://github.com/wso2/docs-apim/pull/3327/commits/769fd2afbc03ac7a5837a4c3fd65abd872859a15
Thanks a lot for pointing this out.